### PR TITLE
mp3 to wav 16khz conversion

### PR DESCRIPTION
**Pull request by kobemoerman**

### CODE REVIEW

File: MusicHatSkill/src/main/kotlin/furhatos/app/musichatskill/SpotifyApiHandler.kt [hunk 1]
```diff
@@ -7,8 +7,8 @@ import com.adamratzman.spotify.spotifyAppApi
 import com.adamratzman.spotify.utils.Market
 
 class SpotifyApiHandler {
-    private val clientID = ""
-    private val clientSecret = ""
+    private val clientID = "9e0aff0020ab499e80ffb83a337779e3"
```
Comment (ErikChHenriksson) [inline MusicHatSkill/src/main/kotlin/furhatos/app/musichatskill/SpotifyApiHandler.kt]:
```suggestion
    private val clientID = ""
```

File: MusicHatSkill/src/main/kotlin/furhatos/app/musichatskill/SpotifyApiHandler.kt [hunk 2]
```diff
@@ -7,8 +7,8 @@ import com.adamratzman.spotify.spotifyAppApi
...
+    private val clientSecret = "d6fe94867ca74b57bf26627e2fe065f8"
```
Comment (ErikChHenriksson) [inline MusicHatSkill/src/main/kotlin/furhatos/app/musichatskill/SpotifyApiHandler.kt]:
```suggestion
    private val clientSecret = ""
```

File: MusicHatSkill/src/main/kotlin/furhatos/app/musichatskill/SpotifyApiHandler.kt [hunk 3]
```diff
@@ -7,8 +7,8 @@ import com.adamratzman.spotify.spotifyAppApi
 import com.adamratzman.spotify.utils.Market
 
 class SpotifyApiHandler {
-    private val clientID = ""
-    private val clientSecret = ""
```
Comment (ErikChHenriksson) [inline MusicHatSkill/src/main/kotlin/furhatos/app/musichatskill/SpotifyApiHandler.kt]: We should not push the credentials to a public repo. Let's keep these empty in the repo and update them only in our own branches.
In the future, don't git add this file after adding the credentials.

**Review by ErikChHenriksson — CHANGES_REQUESTED** [review]
Nice work! Just remove the credentials, then we can merge.

**Review by kobemoerman — COMMENTED** [review]
Ok I approved your reviews.

**Review by ErikChHenriksson — APPROVED** [review]
Looks great!

**Pull request merged**